src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -168,47 +167,36 @@ void playerShot(Coordinate coor) {
 
         if (aircraftCarrier.covers(coor)) {
             if(aircraftCarrier.armorless)
-            {
                 OneShootShip(aircraftCarrier.length,aircraftCarrier.start,aircraftCarrier.end,"player");
```

Comment:
We should probably move OneShootShip to the Civilian ship class to better follow OO design.